### PR TITLE
koordlet: report CPUBasicInfo, adjust normalized cpu cgroups

### PR DESCRIPTION
**Pull request by saintube**
### Ⅰ. Describe what this PR does

koordlet: support CPU Normalization.

1. Report CPUBasicInfo onto NRT annotations.
2. Adjust cgroups cpu.cfs_quota_us for LS and BE (cpushare) pods according to the normalized ratio.
3. Fix a flaky test in pod_resource_collector.go.

### Ⅱ. Does this pull request fix one issue?

Part of #1570.

### V. Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests
- [x] All checks passed in `make test`

### CODE REVIEW

File: pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go [hunk 1]
```diff
@@ -0,0 +1,319 @@
...
+	}
+
+	if !isPodCPUShare(containerCtx.Request.PodLabels, containerCtx.Request.PodAnnotations) {
+		return nil
+	}
+
+	return p.adjustContainerCFSQuota(containerCtx)
+}
+
+func (p *Plugin) adjustPodCPUShares(podCtx *protocol.PodContext) error {
```
Comment (hormes) [inline pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go:206]: Why adjust cpushare?

File: pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go [hunk 2]
```diff
@@ -0,0 +1,319 @@
...
+		return nil
+	}
+
+	ratio := p.rule.GetCPUNormalizationRatio()
+	if ratio == 1.0 { // no need to change
+		containerCtx.Response.Resources.CFSQuota = originalCFSQuota
+		return nil
+	}
+
+	cfsQuota := int64(math.Ceil(float64(*originalCFSQuota) / ratio))
```
Comment (hormes) [inline pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go:295]: Is it possible for ratio to be 0？

File: pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go (hunk 1 above)
Comment (saintube) [reply pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go]: fixed. only adjust cfs quota

File: pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go (hunk 2 above)
Comment (saintube) [reply pkg/koordlet/runtimehooks/hooks/cpunormalization/cpu_normalization.go]: possible when the user manually updates the ratio. fixed by adjust only if the ratio is lager than 1

**Pull request merged**